client/signup/steps/domains/style.scss

```diff
@@ -232,6 +232,34 @@ body.is-section-stepper .domains__step-content,
 	}
 }
 
+.onboarding {
+	&.domains {
+		&.step-route {
+			padding: 48px 0 0;
+			.step-container {
+				.step-container__header {
+					margin: 36px 0 24px 0;
+					@include break-mobile {
+						margin: 48px 0 40px;
+					}
+				}
+			}
+			.formatted-header__title {
+				font-size: 2rem;
+				@include break-mobile {
+					font-size: 2.75rem;
```

Comment:
At a width of, for example, 890px, now on Stepper the page title is 2.75rem

<img width="900" alt="image" src="https://github.com/user-attachments/assets/c29e73fc-114e-4b56-97ee-3c5d312db5b7">

While on Start we still have 2.25rem

<img width="913" alt="image" src="https://github.com/user-attachments/assets/c064041b-c2a3-4858-9d56-372184e8a5b6">

And we jump to 2.75rem only when
`@media (min-width: 1080px)` 

You could probably use `break-xlarge`

```
$break-xhuge: 1920px;
$break-huge: 1440px;
$break-wide: 1280px;
$break-xlarge: 1080px;
$break-large: 960px;	// admin sidebar auto folds
$break-medium: 782px;	// adminbar goes big
$break-small: 600px;
$break-mobile: 480px;
$break-zoomed-in: 280px;
```